README.md

```diff
@@ -196,14 +196,17 @@ Usage: bats [-cr] [-f <regex>] [-j <jobs>] [-p | -t] <test>...
   <test> is the path to a Bats test file, or the path to a directory
   containing Bats test files (ending with ".bats").
 
-  -c, --count      Count the number of test cases without running any tests
-  -f, --filter     Filter test cases by names matching the regular expression
-  -h, --help       Display this help message
-  -j, --jobs       Number of parallel jobs to run (requires GNU parallel)
-  -p, --pretty     Show results in pretty format (default for terminals)
-  -r, --recursive  Include tests in subdirectories
-  -t, --tap        Show results in TAP format
-  -v, --version    Display the version number
+  -c, --count               Count the number of test cases without running any tests
+  -e, --extra-flags         Extra flags to pass to the bats executor (Options: -T)
```

Comment:
This seems a bit cumbersome, can this be replaced by the top-level `-T` (which works fine in `bats -T test/`)

Reply:
I think that is an artifact of the multiple rewrites this has gone through. I believe we can do away with it now.